File: stepik-union/src/main/java/org/stepik/plugin/refactoring/impl/pycharm/move/MoveHandlerDelegate.java
package org.stepik.plugin.refactoring.impl.pycharm.move;

import org.stepik.plugin.refactoring.move.AbstractMoveHandlerDelegate;
import org.stepik.plugin.refactoring.impl.pycharm.AcceptableClasses;

/**
 * @author meanmail
 */
public class MoveHandlerDelegate extends AbstractMoveHandlerDelegate {
    public MoveHandlerDelegate() {
        addAcceptableClasses(AcceptableClasses.get());
    }
}

File: stepik-union/src/main/kotlin/org/stepik/plugin/refactoring/impl/pycharm/move/MoveHandlerDelegate.kt
package org.stepik.plugin.refactoring.impl.pycharm.move

import org.stepik.plugin.refactoring.impl.pycharm.AcceptableClasses
import org.stepik.plugin.refactoring.move.AbstractMoveHandlerDelegate


class MoveHandlerDelegate : AbstractMoveHandlerDelegate() {
    init {
        addAcceptableClasses(AcceptableClasses.get())
    }
}
